doc/migrations/3.3.0_3.3.0plus.md

```diff
@@ -51,7 +51,44 @@ TODO
 
 ### Riak
 
-TODO
+Changes to Riak schema are backwards compatible with the current MongooseIM release.
+This means that in case of skipping of the migrations, only some new features (namely GDPR data retrival) won't work correctly.
+
+#### Step 1
+
+Please update the Riak schema:
+```bash
+# Set the RIAK_HOST to your Riak HTTP endpoint
+# Set the RIAK_MAM_SCHEMA_PATH to point to new schema path, which
+# by default is: RIAK_MAM_SCHEMA_PATH=tools/mam_search_schema.xml
+curl -v -XPUT $RIAK_HOST/search/schema/mam \
+    -H 'Content-Type:application/xml' \
+    --data-binary @${RIAK_MAM_SCHEMA_PATH}
+```
+
+After that we need to either reload all Riak nodes (restart them) or manually reload the schema on live nodes.
+Reloading of schema of live nodes requires access to Erlang Shell of one of the Riak nodes (any of them).
+The instruction on how to get to Riak's Erlang shell is beyond this guide, but if you manage to get to it, just call:
+
+```erlang
+yz_index:reload(<<"mam">>).
+```
+
+#### Step 2
+
+After the schema is posted and reloaded, all "new" objects will be indexed properly as long they contain 2 new fields: `msg_owner_jid` and `mam_type`.
+The new MongooseIM code will insert both of them for all new MAM entires, but for all existing ones, we need to "update" by adding those fields.
+In order to do that, we need to create a migration script (just pick your favourite scripting/programming language) that will do the following for *each* object in *each* bucket of type `mam_yz` (the object will be referred as `obj`):
+
+* Use [this dedicated script](jid-from-mam-muc-script.md) to convert `obj.packet_register` field value into so called `$SENDER_JID`.
```

Comment:
Use [this dedicated script](jid-from-mam-muc-script.md) to convert the `obj.packet_register` field value into a so called `$SENDER_JID`.